### ISSUE TITLE
Java file icons are broken in IU-162.2228.15

### ISSUE THREAD

**Issue by who** [ChrisRM/material-theme-jetbrains#257]
When I enable Material Theme, all of my java files lose their specific icon (indicating that the file is test/class/interface), and instead a "broken jar" icon is used.

My version info:

```
IntelliJ IDEA 2016.2.5
Build #IU-162.2228.15, built on October 14, 2016
JRE: 1.8.0_112-release-287-b2 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o

```

**Comment by M4R1KU** [comment]
Since I updated from IntelliJ 15.x to 2016.2 the icons only appear for a very short time while a gradle build or a sometimes before the end of the `maven clean install` but then they disappear again.

IntelliJ IDEA 2016.2.5
Build IU-162.2032.8
JRE: 1.8.0_101

**Comment by guoyiteng** [comment]
Same thing happens here. However, icons on my windows laptop are correct but they turn into "broken jar" on my mac.

**Comment by denofevil** [comment]
Fixed via #272
